varia/sass/blocks/list/_editor.scss

```diff
@@ -0,0 +1,7 @@
+.block-library-list {
+	ul,
+	ol {
+		margin: #{map-deep-get($config-global, "spacing", "vertical")} 0;
+		padding-left: calc(2 * #{map-deep-get($config-global, "spacing", "unit")});
```

Comment:
Let’s get rid of the `calc()` here and just use `#{2 * map-deep-get($config-global, "spacing", "horizontal")}`. That way its tied to same the horizontal/x-axis spacing metric used in gutters and a few other areas. 

Same should happen for the frontend here: https://github.com/Automattic/themes/pull/1445/files#diff-2ef3dcf0689f1f654e28f9549dcd415dR5

Reply:
👍